src/components/TodoItem.tsx

```diff
@@ -1,23 +1,35 @@
-import { ChangeEvent } from "react";
+import { ChangeEvent, MouseEvent } from "react";
```

Comment:
`MouseEvent` looks like it's unused, so I think we can remove it :)

Reply:
handled in [77230de](https://github.com/ZachWitzig/todoist/pull/3/commits/77230de0aa48039c04a2d2707327e22d807c5848)